plugins/bash-preexec/bash-preexec.sh

```diff
@@ -0,0 +1,341 @@
+# bash-preexec.sh -- Bash support for ZSH-like 'preexec' and 'precmd' functions.
```

Comment:
I'm thinking of updating it to the latest version of bash-preexec in the master branch.

Reply:
done